cfme/tests/cloud/test_providers.py

```diff
@@ -50,14 +52,15 @@ def test_discovery_cancelled_validation(appliance):
 @test_requirements.discovery
 def test_add_cancelled_validation(request):
     """Tests that the flash message is correct when add is cancelled."""
-    prov = EC2Provider()
+    prov = AzureProvider()
     request.addfinalizer(prov.delete_if_exists)
     prov.create(cancel=True)
     view = prov.browser.create_view(CloudProvidersView)
     view.flash.assert_success_message('Add of Cloud Provider was cancelled by the user')
 
 
 @pytest.mark.tier(3)
+@pytest.mark.uncollectif(version.current_version() > '5.8')
```

Comment:
REQUIRED: we don't have 5.7 anymore, why are we uncollecting it now? or add
uncollectif(version.current_version() > '5.8', reason="?????")